### PR TITLE
clarify need for :80 port suffix

### LINKED ISSUE #19865: Importing image from non-https external registry
##### Version
oc v3.9.0+ba7faec-1
kubernetes v1.9.1+a0ce1bc657
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://****:8443
openshift v3.9.0+191fece
kubernetes v1.9.1+a0ce1bc657
##### Steps To Reproduce
1. Create a private docker registry secret with a regular domain 'example.com'.
2. Use oc import-image --insecure=true example.com/test/web-test --confirm

##### Current Result
The import completed with errors.

Name:                   web-test
Namespace:              handsome
Created:                About a minute ago
Labels:                 <none>
Annotations:            openshift.io/image.dockerRepositoryCheck=2018-05-29T10:13:59Z
Docker Pull Spec:       docker-registry.default.svc:5000/handsome/web-test
Image Lookup:           local=false
Unique Images:          1
Tags:                   1

latest
  tagged from harbor.testos39.com/test/web-test
    will use insecure HTTPS or HTTP connections

  ! error: Import failed (Unauthorized): you may not have access to the Docker image "harbor.testos39.com/test/web-test:latest"
      Less than a second ago
  * harbor.testos39.com/test/web-test@sha256:bb94bf5ea1c2fe99a6276f3cca99daa85eb26d50a674e364e14fe151d4df866e
      About a minute ago

error: tag latest failed: you may not have access to the Docker image "harbor.testos39.com/test/web-test:latest"
##### Expected Result
The import completed with success
##### Additional Information

I follow the origin3.9 doc strictly, but it always return Unauthorized! Finally, I see the  code (https://github.com/openshift/origin/blob/release-3.9/pkg/image/importer/credentials.go#L85) that we should add a explicit port '80' in http registry which is not mentioned in  origin3.9 doc. So why not add it in origin3.9 doc?

### PR DESCRIPTION
**Pull request by bparees**
fixes https://github.com/openshift/origin/issues/19865

### CODE REVIEW

**Comment by bparees** [comment]
@malc0lm fyi

**Comment by bparees** [comment]
@openshift/team-documentation ptal. applicable to all releases.

**Comment by bmcelvee** [comment]
Thanks, @bparees! LGTM

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -970,6 +970,12 @@ After the secret is configured, proceed with creating the new image stream or
 using the `oc import-image` command. During the import process, {product-title}
 will pick up the secrets and provide them to the remote party.
 
+[NOTE]
+====
+When importing from an insecure registry, the registry url defined in the secret must include the `:80` port
```
[the remark this answers is not in the document]
Comment (bparees) [reply dev_guide/managing_images.adoc]: fixed

[the remark this answers is not in the document]
Comment (bmcelvee) [reply dev_guide/managing_images.adoc]: Thanks! :)

**Comment by bmcelvee** [comment]
This PR is adding a note for clarification, and does not require QE.

**Comment by malc0lm** [comment]
@bparees Thanks to clarify this!

**Pull request merged**